addons/l10n_ro/data/account_tax_report_data.xml

```diff
@@ -1,405 +1,901 @@
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
+        <field name="parent_id" ref="account_tax_report_ro_baza_intracom_eu"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_24" model="account.tax.report.line">
-        <field name="name">Baza TVA 24%</field>
-        <field name="tag_name">Baza TVA 24%</field>
-        <field name="code">ROTAX_TVA_colectata_24_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd2" model="account.tax.report.line">
+        <field name="name">2 - BAZA - Regularizări livrări intracomunitare scutite conform art. 294 alin.(2) lit.a) şi d) din Codul fiscal</field>
+        <field name="tag_name">2 - BAZA - Regularizări livrări intracomunitare scutite conform art. 294 alin.(2) lit.a) şi d) din Codul fiscal</field>
+        <field name="code">tax_ro_baza_rd2</field>
         <field name="sequence" eval="3"/>
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
+        <field name="parent_id" ref="account_tax_report_ro_baza_intracom_eu"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_5" model="account.tax.report.line">
-        <field name="name">Baza TVA 5%</field>
-        <field name="tag_name">Baza TVA 5%</field>
-        <field name="code">ROTAX_TVA_colectata_5_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd3" model="account.tax.report.line">
+        <field name="name">3 - BAZA - Livrări de bunuri/prestări de servicii pentru care locul livrării/prestării este în afara României, precum şi livrări intracom. de bunuri, scut. conf. art. 294 alin.(2) lit.b) şi c) din CF, din care: </field>
+        <field name="tag_name">3 - BAZA - Livrări de bunuri/prestări de servicii pentru care locul livrării/prestării este în afara României, precum şi livrări intracom. de bunuri, scut. conf. art. 294 alin.(2) lit.b) şi c) din CF, din care: </field>
+        <field name="code">tax_ro_baza_rd3</field>
         <field name="sequence" eval="4"/>
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
+        <field name="parent_id" ref="account_tax_report_ro_baza_intracom_eu"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_9" model="account.tax.report.line">
-        <field name="name">Baza TVA 9%</field>
-        <field name="tag_name">Baza TVA 9%</field>
-        <field name="code">ROTAX_TVA_colectata_9_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd31" model="account.tax.report.line">
+        <field name="name">3.1 - BAZA - Prestări de servicii intracomunitare care nu beneficiază de scutire in statul membru in care taxa este datorată</field>
+        <field name="tag_name">3.1 - BAZA - Prestări de servicii intracomunitare care nu beneficiază de scutire in statul membru in care taxa este datorată</field>
```

Comment:
It is better to have short tag names as they will be shown in the many2many tags (tax grid) for configuring taxes.  Also, the names itself should not be made very long to avoid to need to scroll horizontally.